docs/content/en/docs/crd-ref/_index.md

```diff
@@ -1,11 +1,12 @@
 ---
-title: API Reference
+title: CRD Reference
 description: Reference information about the KLT CRDs
-weight: 100
+weight: 200
 hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
 ---
 
 This section provides comprehensive reference information about the
+YAML files that create the
```

Comment:
I suggest to keep this consistent with the other description text for the other reference pages.
```suggestion
This section provides comprehensive reference information about the
```